### PR TITLE
Update to the maxq10xx support

### PR DESCRIPTION
**Pull request by anhu**

### CODE REVIEW

**Comment by anhu** [comment]
retest this please

File: wolfcrypt/src/random.c [hunk 1]
```diff
@@ -3913,6 +3913,17 @@ int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
 
     #define USE_TEST_GENSEED
 
+#elif defined(WOLFSSL_MAXQ108X) || defined(WOLFSSL_MAXQ1065)
```
Comment (dgarske) [inline wolfcrypt/src/random.c:3916]: I see this code already in random.c above at line 3829... I think you can remove this

**Review by dgarske — CHANGES_REQUESTED** [review]
Only one small issue

File: wolfcrypt/src/random.c (hunk 1 above)
Comment (anhu) [reply wolfcrypt/src/random.c:3916]: Good call, thanks; Fixed!

**Comment by julek-wolfssl** [comment]
Force pushed to run CI

**Comment by dgarske** [comment]
@anhu I rebased this for you...

**Review by dgarske — APPROVED** [review]

**Comment by anhu** [comment]
All tests pass, reviewed and approved.  Please merge at earliest convenience .

**Review by douzzer — APPROVED** [review]

**Pull request merged**